Editor/CHANGES

```diff
@@ -5,6 +5,7 @@
 -Fixed "Auto Set Origin to Center" checkbox not updating EntityProperties UI after checking
 -Fixed "Auto Adjust Origin" in PhysicsPropertiesComponent for scaled objects
 -Fixed copy-paste at wrong position when using right click popup menu
+-Fixed "Open Existing Scene" link, now the scene directory is also selected
```

Comment:
Hmm I don't think we need that in changelog. "Open Existing Scene" wasn't made public yet (there wasn't release yet) so I don't think we should bother listing that in change log.